src/AutoMerge/cron.jl

```diff
@@ -271,152 +271,158 @@ function cron_or_api_build(api::GitHub.GitHubAPI,
-                pkg, version = parse_pull_request_title(NewVersion(), pr)
-            end
-            pr_age = time_since_pr_creation(pr)
-            this_pr_is_old_enough = pr_is_old_enough(pr_type,
-                                                     pr_age;
-                                                     pkg = pkg,
-                                                     new_package_waiting_period = new_package_waiting_period,
-                                                     new_jll_package_waiting_period = new_jll_package_waiting_period,
-                                                     new_version_waiting_period = new_version_waiting_period,
-                                                     new_jll_version_waiting_period = new_jll_version_waiting_period,
-                                                     pr_author = pr_author,
-                                                     authorized_authors = authorized_authors,
-                                                     authorized_authors_special_jll_exceptions = authorized_authors_special_jll_exceptions)
-            if this_pr_is_old_enough
-                i_passed_this_pr,
-                    passed_pkg_name,
-                    passed_pr_head_sha,
-                    status_pr_type = pr_has_passing_automerge_decision_status(api,
-                                                                              registry,
-                                                                              pr;
-                                                                              auth = auth,
-                                                                              whoami = whoami)
-                if i_passed_this_pr
-                    always_assert(pkg == passed_pkg_name)
-                    always_assert(pr.head.sha == passed_pr_head_sha)
-                    _statuses_good = all_specified_statuses_passed(api,
-                                                                   registry,
-                                                                   pr,
-                                                                   passed_pr_head_sha,
-                                                                   all_statuses;
-                                                                   auth = auth)
-                    _checkruns_good = all_specified_check_runs_passed(api, registry,
-                                                                      pr,
-                                                                      passed_pr_head_sha,
-                                                                      all_check_runs;
-                                                                      auth = auth)
-                    if _statuses_good && _checkruns_good
-                        if pr_has_no_blocking_comments(api, registry, pr; auth = auth)
-                            "Pull request: $(pr_number). "
```

Comment:
Why are there loose strings here? Was this supposed to be an `@info`?

Reply:
Yeah that should be an `@info`.

Reply:
Actually there's an `@info` with exactly those strings a few lines below, so these copies should probably just be deleted.